testclock/clock_test.go

```diff
@@ -4,74 +4,70 @@
 package testclock_test
 
 import (
+	"bytes"
+	"fmt"
 	"sync"
-	gotesting "testing"
+	"testing"
 	"time"
 
-	"github.com/juju/loggo"
-	"github.com/juju/testing"
-	jc "github.com/juju/testing/checkers"
-	gc "gopkg.in/check.v1"
+	qt "github.com/frankban/quicktest"
```

Comment:
Unsure about adding another testing library into the Juju ecosystem..

Reply:
FWIW this library is very much general purpose, rather than Juju-specific, and quicktest was developed by a Canonicaler and is already in use by quite a few Canonical modules.

Reply:
+1 for quicktest, it's simpler than gocheck and actively maintained.

Reply:
I like it too, but we already have multiple testing approaches that are partially implemented. Would rather an addition like this was added after consensus has been reached in a thread to the [Juju discource](https://discourse.jujucharms.com/).